src/current/v23.1/migration-strategy-lift-and-shift.md

```diff
@@ -0,0 +1,138 @@
+
+The storage media you use to export / import from can be intermediate data files or streaming data coming over the network. Options include:
+
+- Local "userdata" storage for small tables (see [`cockroach userdata`](cockroach-userfile-upload.html), [Use a Local File Server](use-a-local-file-server.html)).
+- Cloud blob storage (see [`IMPORT INTO`](import-into.html), [Import Performance Best Practices](import-performance-best-practices.html), [Use Cloud Storage](use-cloud-storage.html)).
+- Direct wire transfers (see [managed migration services](#managed-migration)).
+
+Database features for export/import from the source and target databases can include:
+
+- Tools for exporting from the source database may include: `pg_dump --schema-only` and `COPY`, `mysqldump`, `expdp`, etc.
+- For import into CockroachDB, use [`COPY`](copy-from.html) or [`IMPORT INTO`](import-into.html). For a bulk data migrations, most users should use `IMPORT INTO` because the tables will be offline anyway, and `IMPORT INTO` can [perform the data import much faster](import-performance-best-practices.html) than `COPY`.
+
+Decide which of the options above will meet your requirements while resulting in a process that is [restartable](#restartable) and [scalable](#scalable).
+
+<a name="restartable"></a>
+
+### Design a restartable process
+
+To have a higher chance of success, design your data migration process so it can be stopped and restarted from an intermediate state at any time during the process. This will help minimize errors and avoid wasted effort.
+
+Keep the following requirements in mind as you design a restartable import/export process:
+
+- Bulk migrate data in manageable size batches for your source and target systems.
+    - This is a best practice. If something happens to the target cluster during import, the amount of wasted work will be minimized.
+- Implement progress/state keeping with process restart capabilities.
+- Make sure your export process is idempotent: the same input to your export process should return the same output data.
+- If possible, export and import the majority of your data before taking the source database down. This can ensure that you only have to deal with the incremental changes from your last import to complete the migration process.
+
+<a name="scalable"></a>
+
+### Design a scalable and performant process
+
+Once your process is [restartable and resilient to failures](#design-a-restartable-process), it's important to also make sure it will scale to the needs of your data set. The larger the data set you are migrating to CockroachDB, the more important the performance and scalability of your process will be.
+
+Keep the following requirements in mind:
+
+- Schema and data should be imported separately.
+- Your process should handle multiple files across multiple export/import streams concurrently.
+  - For best performance, these files should contain presorted, disjoint data sets.
```

Comment:
Should that be "disjointed" or is this a term I don't know?

Reply:
ya it's a sets term: https://en.wikipedia.org/wiki/Disjoint_sets